### PR TITLE
chore: add Docker development environment

### LINKED ISSUE #248: Docker environment
It could be nice to have a simple Dockerfile to speed up the local setup

### PR DESCRIPTION
**Pull request by bfabio**
Fix #248

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,13 +10,17 @@ Ogni commento o proposta relativa all'evoluzione dei kit può essere fatta utili
...
-    $ yarn install
-    $ bundle install
-    $ bundle exec jekyll serve
+1. Crea l'immagine Docker
+   ```console
+   docker build . -t designers-italia-it
+   ```
 
-In caso di problemi con l'installazione di rmagick 4.1.2 su recenti OSX, [quest pagina](https://stackoverflow.com/questions/41647979/imagemagick-7-with-rmagick-2-16-on-macos-sierra-cant-find-magickwand-h) potrebbe essere d'aiuto.
+1. Esegui l'ambiente di sviluppo
```
Comment (libremente) [inline README.md:20]:
```suggestion
2. Esegui l'ambiente di sviluppo
```

**Review by libremente — CHANGES_REQUESTED** [review]
Thanks @bfabio! Please also add a line in the README with the localhost:4000 indication :)

**Comment by bfabio** [comment]
> Thanks @bfabio! Please also add a line in the README with the localhost:4000 indication :)

Right, good catch!

**Review by libremente — APPROVED** [review]
LGTM! :rocket:

**Pull request merged**